warehouse/core/src/main/java/datawave/util/ColumnSetUtil.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Taken from org.apache.accumulo.core.iteratorsImpl.conf.ColumnSet.
+ * ColumnSet was not part of Accumulo's public API, so the parts that were needed were copied here.
+ */
+
+package datawave.util;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import org.apache.accumulo.core.util.Pair;
+import org.apache.hadoop.io.Text;
+
+public class ColumnSetUtil {
+
+    public static String encodeColumns(Text columnFamily, Text columnQualifier) {
+        StringBuilder sb = new StringBuilder();
+
+        encode(sb, columnFamily);
+        if (columnQualifier != null) {
+            sb.append(':');
+            encode(sb, columnQualifier);
+        }
+
+        return sb.toString();
+    }
+
+    public static Pair<Text,Text> decodeColumns(String columns) {
```

Comment:
Lets also avoid using Pair here (part of private api).  Use AbstractMap.SimpleEntry instead please